src/components/Header.css

```diff
@@ -0,0 +1,82 @@
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 38px;
+  padding: 15px 20px;
+  background-color: #fff;
+}
+
+.header {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  height: 45px;
+}
+
+.header__link {
+  font-family: sans-serif, Montserrat;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  cursor: pointer;
+  font-weight: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  font-size: 13px;
+  height: 16px;
+  margin: 20px;
+  opacity: 0.5;
+}
+
+.header__logo {
+  margin-left: 0;
+  height: 30px;
+  font-size: 26px;
+  letter-spacing: normal;
+  color: #0290ff;
+  font-weight: bold;
+  opacity: 1;
+}
+
+.header__book_link {
+  opacity: 1;
+}
+
+.user {
+  width: 25px;
+  height: 25px;
+  padding: 10px;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-self: center;
+}
+
+.user-logo {
+  background-image: url('../images/user.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: block;
+  width: 15px;
+  height: 15px;
+}
```

Comment:
Let's add a `margin: 0 auto;` CSS rule to center the logo horizontally inside div with a circular border.